src/app.js

```diff
@@ -2,137 +2,243 @@
 /* eslint-disable @typescript-eslint/unbound-method */
 /* eslint-disable @typescript-eslint/no-var-requires */
 // Required imports
-const { ApiPromise, WsProvider } = require('@polkadot/api');
+const { Keyring, ApiPromise, WsProvider } = require('@polkadot/api');
 const PlugRuntimeTypes = require('@plugnet/plug-api-types');
-const Keyring = require('@polkadot/keyring');
-const testingPairs = require('@polkadot/keyring/testingPairs');
+const { Api } = require('@cennznet/api');
 const cli = require('../src/cli.js');
 const selector = require('../src/selector.js');
+const addStaking = require('../src/stakingSetup');
+const { makeTransaction } = require('../src/transaction')
+const { sleep } = require('../src/utils')
 require('console-stamp')(console, 'HH:MM:ss.l');
+let log = require('console-log-level')({ level: 'info' });
+let createStashAccounts = require('../src/stakingSetup')
 
 const APP_SUCCESS = 0;
-const APP_FAIL_TRANSACTION_REJECTED = 1;
```

Comment:
Shouldnt really move this out of this file. These are the error codes the script will return to the terminal so they belong here.
A good approach is to just throw the error in the transaction module and then catch the error in this module and then add the error code.

Reply:
`while(timed_out == false) {
      if (hash != null) {
        clearTimeout(timer);
        break;
      }
      if (transaction_error != null) {
        throw [APP_FAIL_TRANSACTION_REJECTED, id_string + `Transaction rejected: ${transaction_error}`];
      }
      await sleep(sleep_ms);
    }`

this part in transaction.js throw the error code and should be catched when main function executes. Or maybe I'm wrong?

Reply:
You are correct,  but we could just do:
```
throw (id_string + `Transaction rejected: ${transaction_error}`); 
```